src/components/global/Footer.jsx

```diff
@@ -1,8 +1,208 @@
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
```

Comment:
Container isn't being used anywhere

Reply:
Pretty sure the entire component should be in a `Container` but I ran into issues using it that worked with the `Box`. As far as I am aware, to MUI standards it should be in a container not a box so I am trying to get that switched and working